benchmarks/AclBench.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
Let's make sure this directory, as well as the phpbench config, are listed as `export-ignore` in `.gitattributes`